backfill.sql

```diff
@@ -29,6 +29,10 @@
 * of that table in certain ranges faster. (ie `CREATE INDEX ON cpu_temp(time);`)
 */
 
+---- Backfill operation enum type to define the behavior in case of conflicting rows
+DROP TYPE IF EXISTS BackfillOnConflict;
+CREATE TYPE BackfillOnConflict AS ENUM ('DoNothing', 'Update', 'Error');
```

Comment:
A couple comments: 
1) I think I'd have the names here be 'NOTHING', 'UPDATE', 'RESTRICT' just to keep them consistent with PG naming.
2) I'm not sure if I'd actually use an ENUM here or if I'd just pass in text to the function and then compare it there and error if it's not one of the three, mainly because I think ENUMs are case sensitive, (so I'd actually compare UPPER(foo) or whatever) and I think it'd be weird to have to match the case so closely.